charts/longhorn-upgrade-responder/0.1.2/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: A Helm chart for Kubernetes
+name: upgrade-responder
+version: v0.1.1
```

Comment:
Source: https://github.com/longhorn/upgrade-responder/blob/v0.1.2/chart/Chart.yaml

Reply:
They've updated to v0.1.2 in this commit: https://github.com/longhorn/upgrade-responder/commit/e544c16910d7235b8ae8c8aeb6061533651f4b83